utils/generic.py

```diff
@@ -278,7 +278,7 @@ def vep_or_lookup_vep(ht, reference_vep_ht=None, reference=None, vep_config=None
     :return: VEPped Table
     :rtype: Table
     """
-    from gnomad_hail.resources.basics import vep_config_path, context_ht_path
+    from gnomad_qc.resources.basics import vep_config_path, context_ht_path
```

Comment:
we need this to work without importing `gnomad_qc` resources. the VEP configs should defintely stay in the `gnomad_hail` repo.